website/source/docs/state/environments.html.md

```diff
@@ -114,7 +114,7 @@ aren't any more complex than that. Terraform wraps this simple notion with
 a set of protections and support for remote state.
 
 For local state, Terraform stores the state environments in a folder
-`terraform.state.d`. This folder should be committed to version control
+`terraform.state.d`. This folder should not be committed to version control
```

Comment:
It actually should be, this is not a typo! If you're using local-only state, it should be committed to version control. If you don't want state in version control then you should be using remote backends.